cudax/test/containers/async_buffer/access.cu

```diff
@@ -95,12 +95,12 @@ C2H_CCCLRT_TEST("cudax::async_buffer access and stream", "[container][async_buff
 
     {
       cudax::stream other_stream{cuda::device_ref{0}};
```

Comment:
are we not testing `set_stream` anywhere? I would have expected that we would have t change some test

Reply:
🙈